crates/viewer/re_space_view_spatial/src/picking.rs

```diff
@@ -67,10 +67,18 @@ pub struct PickingContext {
     /// Cursor position on the renderer canvas in pixels.
     pub pointer_in_pixel: glam::Vec2,
 
-    /// Cursor position in the 2D space coordinate system.
+    /// Cursor position in the UI coordinates after panning & zooming.
     ///
-    /// For 3D spaces this is equal to the cursor position in pixel coordinate system.
-    pub pointer_in_space2d: glam::Vec2,
+    /// Note that for 2D spaces these are the final 2D space coordinates.
+    ///
+    /// As of writing, for 3D spaces this is equal to [Self::pointer_in_ui],
+    /// since we don't allow panning & zooming after perspective projection.
+    pub pointer_in_ui_after_pan_and_zoom: glam::Vec2,
+
+    /// Apply pan & zoom to ui coordinates.
+    ///
+    /// See also [`Self::pointer_in_ui_after_pan_and_zoom`].
+    pub ui_pan_and_zoom_from_ui: egui::emath::RectTransform,
```

Comment:
I think we should come up with a better name for this space. "ui_pan_and_zoom" is overlong and complicated.

After zooming and panning, it is not really `ui` related any more. Maybe call it "flat_space" ?

Reply:
was struggling with this quite a bit. Before it said "space2d" but this really confused me upon revisiting the code which is why I went with the overly verbose thing.
Something I had in here in the meantime was `camera_plane` because if this is 3D then these are coordinates on the camera plane which then is the thing we're panning and zooming (imagine what happens with pan & zoom on the arkit 2D scene).
Maybe a better name than that still would be `camera_ui`-space (+ doc on why it's name this way). What do you think?

Reply:
I like `camera_plane`, since we already have a `ui` coordinate system, so better to keep the names as distinct as possible